public/locales/he/translation.json

```diff
@@ -105,5 +105,11 @@
       "compared to national average": "במקטע לעומת ממוצע ארצי בכבישים מאותו הסוג"
     },
     "No data": "אין נתונים להצגה"
+  },
+  "mapDialog": {
+    "searchButton": "חיפוש",
+    "cancelButton": "ביטול",
+    "chosenSegment": "המקטע שנבחר: ",
```

Comment:
same here
spaces can be done with padding or &nbsp;
should not be part of i18n files (as this is easy to break by mistake)